docs/additional-features/stress-testing.md

```diff
@@ -8,24 +8,46 @@ title: Network stress testing
 This guide assumes that:
 
 - You have a working Polygon-SDK network up and running
-- Both your JSON-RPC endpoints are reachable
-- For each of the sender accounts you use, its private key must be saved as an environment variable with the following format:
-  - PSDK_ADDRESS=PRIVATE_KEY
-- Each sender account must have the required funds on it to execute a single loadbot run
+- Both your JSON-RPC and GRPC endpoints are reachable
 
-e.g. to export the variable :
+## Overview
+
+The Polygon SDK loadbot is a helper that is meant to stress test the Polygon SDK cluster.
+
+Currently, it supports 2 modes:
+
+- `Transfer` - mode that does stress testing using fund-transfer transactions. **[Default]**.
+- `Deploy` - mode that deploys specified smart contracts with each transcation.
```

Comment:
I think `transfer` and `deploy` (which are same values to ones used in command) are better in order not to make users confused. How do you think?

Reply:
You're right 👍 

Fixed:

230c12e